docs/BEMAN_STANDARD.md

```diff
@@ -108,7 +108,8 @@ The top-level of a Beman library repository must consist of `CHANGELOG.md`, `CMa
 `LICENSE`, and `README.md` files.
 
 **[TOPLEVEL.CHANGELOG]** REQUIREMENT: There must be a `CHANGELOG.md` file at the repository's root
-that describes the changes in each version of the library.
+that describes the high level changes in each version of the library (e.g., library status change,
+new paper implementation addition, paper implementation removal etc). 
```

Comment:
Do the examples from below need to be moved up here?

Reply:
I don't think so.

I'm OK to do a follow-up PR to fix that if you think otherwise.

He is my motivation (I think alo said that when I did the initial changelog PR):
- the same thing we do for README.md
<img width="1173" alt="image" src="https://github.com/user-attachments/assets/18ac8435-34aa-4ca6-b517-d8e108b27d4c" />
and then we have like a "README format section"
<img width="1104" alt="image" src="https://github.com/user-attachments/assets/e00fba73-c8e7-4537-8037-5e5ced3a7c5b" />
- Think examples are more appropiate for a rule CHANGELOG.FORMAT, as it has multiple details (like the Keep a Changelog convention):
<img width="889" alt="image" src="https://github.com/user-attachments/assets/d0e60601-e481-4e9f-ac4d-e719b53dff58" />

Reply:
Let me know if you want a follow-up for this topic. I'm OK anyways, it's a style topic.